test/models/retinanet/RETINANET_README.md

```diff
@@ -0,0 +1,5 @@
+To test the RetinaNet you need to install the retinaNet env:
+
+1. Create a virtual environment that uses python 3.8 (via Poetry or Conda) and activate it
+2. pip install -r requirements.txt
+3. python -m pip install 'git+https://github.com/facebookresearch/detectron2.git'
```

Comment:
It won't work man we need to run it on CI.
Add skip on the test for now and we will get back to it later

Reply:
skipped the test and changed PR to draft